beacon_node/network/src/sync/block_lookups/common.rs

```diff
@@ -171,3 +175,52 @@ impl<T: BeaconChainTypes> RequestState<T> for BlobRequestState<T::EthSpec> {
         &mut self.state
     }
 }
+
+impl<T: BeaconChainTypes> RequestState<T> for CustodyRequestState<T::EthSpec> {
+    type VerifiedResponseType = Vec<CustodyDataColumn<T::EthSpec>>;
+
+    fn make_request(
+        &self,
+        id: Id,
+        // TODO(das): consider selecting peers that have custody but are in this set
+        _peer_id: PeerId,
```

Comment:
Is it worth trying to retrieve from this peer at all? It could send an attestation with unknown block but may not have be custodying the same columns as our node.